backend/rorapp/views/waitlist_entry.py

```diff
@@ -20,7 +21,7 @@ def create(self, request, *args, **kwargs):
         isEntryPresent = WaitlistEntry.objects.filter(email=data["email"])
 
         if isEntryPresent:
-            return Response({}, status=status.HTTP_200_OK)
+            return Response({"email": data["email"], "entry_date": timezone.now()}, status=status.HTTP_201_CREATED)
```

Comment:
This is perfect, well done! I tested it locally and the real/fake responses are indistinguishable.